streams/src/main/scala/tofu/streams/Evals.scala

```diff
@@ -0,0 +1,28 @@
+package tofu.streams
+
+import cats.syntax.foldable._
+import cats.{Applicative, Foldable, Monad}
+
+trait Emits[F[_]] {
```

Comment:
Why is this better than Alternative?

Reply:
@Odomontois, I can't remember any existing abstraction covering case like `def emits[A](as: List[A]): F[A]`.

Reply:
I think Oleg implied pure+combineK

Reply:
@Odomontois, it turned out there is no lawful implementation of `Alternative` for `Stream`.

Reply:
@oskin1 Actually that's interesting. What laws are broken?